Gemfile

```diff
@@ -2,5 +2,15 @@ source 'https://rubygems.org'
 
 gemspec
 
+def gems(*gems)
+  gems.each { |g| gem(g) }
+end
+
 gem 'addressable'
 gem 'faraday', '~> 0.11.0'
+
+group :development do
+  gems "bundler-audit",
```

Comment:
"bundler-audit" is not needed for Rubocop, but I'm adding it anyway. In one of the next steps we will/might add it to CI as separate step.